cmd/trivy/main.go

```diff
@@ -117,6 +117,11 @@ OPTIONS:
 			Value: utils.DefaultCacheDir(),
 			Usage: "use as cache directory, but image cache is stored in /path/to/cache/fanal",
 		},
+		cli.StringFlag{
+			Name:  "ignorefile",
+			Value: ".trivyignore",
+			Usage: "specify .trivyignore file (default: ./.trivyignore)",
```

Comment:
```suggestion
			Usage: "specify .trivyignore file",
```
The default value is filled automatically by the library.